cmd/crane-agent/app/options/option.go

```diff
@@ -32,7 +42,10 @@ func (o *Options) Validate() error {
 // AddFlags adds flags to the specified FlagSet.
 func (o *Options) AddFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.HostnameOverride, "hostname-override", "", "Which is the name of k8s node be used to filtered.")
-
 	flags.StringVar(&o.RuntimeEndpoint, "runtime-endpoint", "unix:///var/run/dockershim.sock", "The runtime endpoint, default to docker: unix:///var/run/dockershim.sock, containerd: unix:///run/containerd/containerd.sock.")
+	flags.Bool("enable-profiling", false, "Is debug/pprof endpoint enabled, default: false")
+	flags.StringVar(&o.BindAddr, "bind-address", "0.0.0.0:8081", "The address the agent binds to,default: 0.0.0.0:8081.")
```

Comment:
can you state what's the purpose of the address binding? for pprof or health check?